src/layers/core/path-layer/path-layer-vertex.glsl

```diff
@@ -1,71 +1,197 @@
 #define SHADER_NAME path-layer-vertex-shader
 
-attribute float directions;
 attribute vec3 positions;
-attribute vec3 leftDeltas;
-attribute vec3 rightDeltas;
 
-attribute vec4 colors;
-attribute vec3 pickingColors;
+attribute vec3 instanceStartPositions;
```

Comment:
I pushed a new commit that simplifies the vertex shader. Will run some perf test tomorrow.